### PR TITLE
[Spark-5717] [MLlib] add stop and reorganize import

### PR DESCRIPTION
**Pull request by hhbyyh**
Trivial. add sc stop and reorganize import
https://issues.apache.org/jira/browse/SPARK-5717

### CODE REVIEW

**Comment by hhbyyh** [comment]
The ut failure seems irrelevant.
Jenkins, retest this please.

File: examples/src/main/scala/org/apache/spark/examples/mllib/LDAExample.scala [hunk 1]
```diff
@@ -21,15 +21,12 @@ import java.text.BreakIterator
...
-import scopt.OptionParser
-
 import org.apache.log4j.{Level, Logger}
-
-import org.apache.spark.{SparkContext, SparkConf}
 import org.apache.spark.mllib.clustering.LDA
 import org.apache.spark.mllib.linalg.{Vector, Vectors}
 import org.apache.spark.rdd.RDD
-
+import org.apache.spark.{SparkConf, SparkContext}
```
Comment (srowen) [inline examples/src/main/scala/org/apache/spark/examples/mllib/LDAExample.scala]: The Spark imports were correctly ordered already. I think the scopt import could go after the log4j import, if we're matching style of other imports.

**Comment by srowen** [comment]
Yes it's a small thing but the examples should call `stop()`. I can merge if you'll move the imports accordingly.

File: examples/src/main/scala/org/apache/spark/examples/mllib/LDAExample.scala (hunk 1 above)
Comment (hhbyyh) [inline examples/src/main/scala/org/apache/spark/examples/mllib/LDAExample.scala]: OK.

**Comment by srowen** [comment]
jenkins, retest this please

**Pull request closed without merging**